docs/ox_core/Vehicle/Server/index.md

```diff
@@ -12,6 +12,15 @@ local vehicle = Ox.GetVehicle(entity)
 print(json.encode(vehicle, { indent = true }))
 ```
 
+## Ox.GetVehicleFromNetId
+
+Return an instance of CVehicle for the given entity, this time using it's netId.
```

Comment:
"this time using it's netId." aside from being grammatically incorrect just reads.. oddly?

Going to be moving away from the "instance of Class" language too since most people won't get it.